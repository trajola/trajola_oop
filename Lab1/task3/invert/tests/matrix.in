3	4	8
2.4	-1	11
1	-3.2	0