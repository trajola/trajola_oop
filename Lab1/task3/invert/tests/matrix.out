          0.366         -0.266          0.541
          0.114         -0.083         -0.144
         -0.069          0.141         -0.131
